docs/resources.rst

```diff
@@ -1,45 +1,49 @@
+#########
 Resources
----------
+#########
 
-General
-~~~~~~~
+General Resources
+=================
 
-* `Ethereum <https://ethereum.org>`_
+* `Ethereum.org Developer Portal <https://ethereum.org/en/developers/>`_
+* `Ethereum StackExchange <https://ethereum.stackexchange.com/>`_
+* `Solidity Portal <https://soliditylang.org/>`_
+* `Solidity Changelog <https://github.com/ethereum/solidity/blob/develop/Changelog.md>`_
+* `Solidity Source Code on GitHub <https://github.com/ethereum/solidity/>`_
+* `Solidity Language Users Chat <https://gitter.im/ethereum/solidity/>`_
+* `Solidity Compiler Developers Chat <https://gitter.im/ethereum/solidity-dev/>`_
```

Comment:
Is there a matrix version for these instead of gitter? They might deprecate the gitter version at some point

Reply:
changed to Matrix